CMSgov/mint-app
# [EASI-3215] Allow multiple roles for collaborators

# EASI-3215

## Changes and Description

- Added flyway migration updating team_role to team_roles array
- Updated gql schema
- Updated resolvers
- Updated data model



## How to test this change

- Bring the systems up
- Create a plan collaborator using Postman
- Ensure the expected roles exist within Postman
- Update a plan collaborator's roles using Postman
- Delete the final Model Lead plan collaborator using Postman
  - This should throw an error

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
